Unity-Technologies/Graphics
# Fix issue where node inputs would misalign from their ports

### **Please read**
Fix for https://fogbugz.unity3d.com/f/cases/1224480/. This recurring bug keeps coming back due to an out-of-date optimization in the NodeView structure. I removed the optimization because the UIR renderer doesn't need it, and simplified the node architecture to ensure node inputs stay with their ports.

### Checklist for PR maker

---
### Testing status

**Manual Tests**: What did you do?
- Other: 
Created lots of nodes, re-ordered ports on CFN and Subgraphs, expanded/collapsed node port and preview sections.

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)
None; it's all UI. Did verify existing port tests work.

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline
